mmv1/products/privateca/api.yaml

```diff
@@ -77,6 +81,12 @@ objects:
         required: true
         input: true
         url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: pool
+        description: The name of the CaPool this Certificate Authority belongs to.
+        required: true
+        input: true
+        url_param_only: true
       - !ruby/object:Api::Type::String
         name: 'name'
         description: |
```

Comment:
(misplaced comment, should be on line 110)

'tier' is no longer an input field on a CA. it gets set on the CaPool resource and the service copies it to an OUTPUT_ONLY field on each CA within that pool.